plugins/mcreator-core/jsontriggers/entity_hurt_player.json

```diff
@@ -0,0 +1,42 @@
+{
+  "args0": [
+    {
+      "type": "input_value",
+      "name": "minDamage",
+      "check": "Number"
+    },
+    {
+      "type": "input_value",
+      "name": "maxDamage",
+      "check": "Number"
+    },
+    {
+      "type": "field_data_list_selector",
+      "name": "source_entity",
```

Comment:
We usually call this sourceentity

![image](https://github.com/MCreator/MCreator/assets/16374228/a8a0a118-11e0-4f48-8b81-e3420024d238)

Reply:
Renamed it to sourceentity